lsif/src/server/backend/backend.ts

```diff
@@ -465,10 +465,10 @@ export class Backend {
         // First get all LSIF reference result locations for the given position.
         const locationSet = await database.references(cursor.path, cursor.position, ctx)
 
-        // Search the references table of the current dump. This search is necessary, but may be
-        // un-intuitive. A 'Find References' operation on a reference should also return references
-        // to the definition. These are not necessarily fully linked in the LSIF data. This method
-        // returns a cursor if there are reference rows remaining for a subsequent page.
+        // Search the references table of the current dump. This search is necessary because we
+        // want a 'Find References' operation on a reference to also return references to the 
+        // the governing definition, and those may not be fully linked in the LSIF data.
```

Comment:
```suggestion
        // want a 'Find References' operation on a reference to also return references to
        // the governing definition, and those may not be fully linked in the LSIF data.
```

This one was my fault, sorry.

Reply:
They're on different lines so it's easy to miss :p

Reply:
At least we get multi-line comments now!